src/pages/virtual-machines/virtual-machines-list/VirtualMachinesList.tsx

```diff
@@ -25,35 +28,58 @@ import { Menu } from 'primereact/menu';
 import { RestoreModal } from 'pages/virtual-machines/modal/RestoreModal';
 import HeaderTable from '../../../components/table/HeaderTable';
 import { backupsService } from '../../../services/backups-service';
-import { resetTaskAction } from '../../../store/mount-backup-modal/actions';
+import { resetTaskAction } from 'store/mount-backup-modal/actions';
 import { NoActiveRulesIcon } from 'components/modal/BackupModal/NoActiveRulesIcon';
 
 const VirtualMachinesList = () => {
   const dispatch = useDispatch();
   const history = createBrowserHistory();
-  const [globalFilter, setGlobalFilter] = useState(null);
   const [actionsElement, setActionsElement] = useState(null);
 
+  const [apiRequestData, setApiRequestData] = useState({
```

Comment:
This shouldn't be part of this component because wee don't want to add it to every other component

Reply:
@sebastsliwin97 should we add it to the store?

Reply:
@krakowskiprzem table component should provide this params